src/reporting/lambda/custom-resource/quicksight/index.ts

```diff
@@ -1113,18 +1114,32 @@ const createOrUpdateRefreshSchedule = async (quickSight: QuickSight, commonParam
       datasetId,
       scheduleId,
     });
-    await quickSight.createRefreshSchedule({
-      AwsAccountId: commonParams.awsAccountId,
-      DataSetId: datasetId,
-      Schedule: {
+
+    let schedule: RefreshSchedule;
+    if (refreshInterval === RefreshInterval.HOURLY) {
```

Comment:
assign a variable to  `RefreshInterval.DAILY` or `HOURLY`, then share the code to create schedule